crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort.rs

```diff
@@ -126,9 +128,28 @@ fn compare_classes(a: &ClassInfo, b: &ClassInfo) -> Ordering {
 }
 
 /// Sort the given class string according to the given sort config.
-pub fn sort_class_name(class_name: &TokenText, sort_config: &SortConfig) -> String {
+/// ignore_prefix and ignore_suffix are used to ignore the first and last class respectively.
+pub fn sort_class_name(
+    class_name: &TokenText,
+    sort_config: &SortConfig,
+    ignore_prefix: bool,
+    ignore_suffix: bool,
+) -> String {
     // Obtain classes by splitting the class string by whitespace.
-    let classes = class_name.split_whitespace().collect::<Vec<&str>>();
+    let mut classes_iter = class_name.split_whitespace();
+    let class_str_prefix = if ignore_prefix {
+        classes_iter.next()
+    } else {
+        None
+    };
```

Comment:
```suggestion
    let class_str_prefix = ignore_prefix.and_then_some(classes_iter.next());
```

Reply:
https://github.com/hangaoke1/biome/blob/perf/use_sorted_classes/crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort.rs#L131
i add comment in this pr 2 weeks ago